### DISCUSSION TITLE
in production mode, Why tauri cannot access its own spawn serve

### DISCUSSION THREAD

**Discussion by bingryan** [tauri-apps/tauri#6731] [Q&A]
I use `tauri:: async_ Runtime:: spawn` launched an axum serve (port: 9999) and created a function for uploading and previewing images

In the development mode (pnpm tauri dev), all functions are normal, but the dmg packaged with build (pnpm tauri build), upload and display functions are no longer available

But  in production mode I tried to access: axum's serve can be accessed, and I would like to ask how to configure the production mode to meet my needs

**Comment by Disorrder** [comment]
Hey, do you remember how did you solve this?
